### PR TITLE
Update migrations.md

### PR DESCRIPTION
**Pull request by robbiehanson**
Attempting to explain the solution from issue #689.
That the options can also specify options (such as gasPrice) when sending the transaction.

### CODE REVIEW

**Comment by robbiehanson** [comment]
Link to [Issue #689](https://github.com/trufflesuite/truffle/issues/689) -  Fine-tuned gasPrices during migration

File: public/docs/getting_started/migrations.md [hunk 1]
```diff
@@ -172,6 +172,16 @@ Deploy a specific contract, specified by the contract object, with optional cons
 
 You can optionally pass an array of contracts, or an array of arrays, to speed up deployment of multiple contracts. Additionally, the last argument is an optional object that observes a single key, `overwrite`. If `overwrite` is set to `false`, the deployer won't deploy this contract if one has already been deployed. This is useful for certain circumstances where a contract address is provided by an external dependency.
 
+```javascript
+deployer.deploy(Migrations, {overwrite: false});
+```
+
+The options can also be used to specify the parameters when sending the transaction:
```
Comment (gnidan) [inline public/docs/getting_started/migrations.md]: Would you mind updating the line above that mentions a “single key”, since `gas` and `from` are also accepted?

File: public/docs/getting_started/migrations.md [hunk 2]
```diff
@@ -172,6 +172,16 @@ Deploy a specific contract, specified by the contract object, with optional cons
 
 You can optionally pass an array of contracts, or an array of arrays, to speed up deployment of multiple contracts. Additionally, the last argument is an optional object that observes a single key, `overwrite`. If `overwrite` is set to `false`, the deployer won't deploy this contract if one has already been deployed. This is useful for certain circumstances where a contract address is provided by an external dependency.
```
Comment (tcoulter) [inline public/docs/getting_started/migrations.md]: @bmmpxf Can you change 
> that observes a single key `overwrite`

To: 

> that can include the key `overwrite` as well as other transaction parameters

so full sentence: 

> Additionally, the last argument is an optional object that can include the key `overwrite` as well as other transaction parameters.

File: public/docs/getting_started/migrations.md (hunk 1 above)
Comment (tcoulter) [reply public/docs/getting_started/migrations.md]: @bmmpxf I'd reword this as the parameters have already been referenced, then link out to somewhere else if the reader wants more information. I did a quick search of the Truffle docs and couldn't find an example, so perhaps link to the `truffle-contract` readme.

**Comment by bmmpxf** [comment]
Hi @robbiehanson. Thanks for the suggestions, this is all good stuff. I've taken your PR, made some changes as suggested by @tcoulter, and committed with https://github.com/trufflesuite/trufflesuite.com/commit/9cc758ca8208f6bb874853b588cbe09a59ae3374. Please take a look and make sure that I've addressed your concerns. And if you find other areas where we could be more detailed, please let us know. Thanks!
